src/pages/patientView/copyNumberAlterations/column/CohortColumnFormatter.tsx

```diff
@@ -72,4 +88,52 @@ export default class CohortColumnFormatter
             );
         });
     }
+
+    private static getGisticValue(data:DiscreteCopyNumberData, gisticData:IGisticData)
+    {
+        const gistic = gisticData[data.entrezGeneId];
+        let summary:IGisticSummary|undefined;
+
+        if (gistic)
+        {
+            // here we are assuming that we have at most 2 values in the GisticSummary array:
+            // one for amp === true, and one for amp === false
+            summary = gistic.find((gs:IGisticSummary) => {
+                // alteration === 2 => amplified
+                // otherwise => not amplified (deleted)
+                return gs.amp === (data.alteration === 2);
+            });
+        }
+
+        return summary === undefined ? null : summary;
+    }
+
+    private static makeGisticIcon(qValue:number, peakGeneCount:number)
+    {
+        return (
+            <DefaultTooltip
+                placement="right"
+                overlay={CohortColumnFormatter.getGisticTooltip(qValue, peakGeneCount)}
```

Comment:
Good point. In that case, we should also revisit the existing tool tip generating mechanism in general.